PascalABC.NET.Compiler/PascalABC.NET.Compiler.nuspec

```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+
+    <metadata>
+        <id>FVNever.PascalABC.NET.Compiler</id>
+        <version>$version$</version>
+        <authors>ForNeVeR</authors>
+        <license type="expression">LGPL-3.0-only</license>
```

Comment:
Unrelated: Why Nuspec and not properties on the csproj?

Reply:
Because I've copy-pasted this from a known working example. Will try converting to `.proj`, that's a good suggestion.

Reply:
Eventually decided against that.

We have nice responsibility separation:
- `.proj` file downloads the files (and so has file hashes and download tasks)
- `.nuspec` file packs the files

I've thought about merging this into the `.proj` file, but I think it will be too messy. Consider this:
```
<LicenseArtifact Include="License_en.txt" Sha256Hash="…" PackPath="[???]" />
```

We have three paths for this `LicenseArtifact`: path on disk (`obj/doc/License_en.txt`), virtual path in the project (not necessary really, but I'm not sure it's possible to have an item without `Include=`), download URL.

I found no way to put every piece of necessary information without too much duplication.

And the stuff with `.nuspec` works nicely.

Reply:
Did you look at https://github.com/devlooped/nugetizer I did not try. Just want to mention that this is maybe good solution

Reply:
I did, and that's too much magic for me to handle.